coreax/metrics.py

```diff
@@ -25,12 +25,12 @@
 # Support annotations with | in Python < 3.10
 from __future__ import annotations
```

Comment:
Given you have largely rewritten this module, would it make sense to ditch this line and add quotes around annotations that are offending in 3.9?

Reply:
As I understand, quoting the annotations would be functionally the same as using this import. The alternative would be using the `Union[..., ...]` type, but we don't use this anywhere else in the codebase.

Reply:
In Vanguard, we made the decision that future imports for annotations are a bad idea due to their ability to hide all sorts of evils. Quoting instead provides the exception/ignore on a per-annotation basis rather than per-file. Coreax should move towards this but I'm happy to leave it for now to maintain consistency with the rest of the Coreax codebase.